setup.py

```diff
@@ -52,6 +52,18 @@
     'scipy>=0.18',
     'netCDF4',
     'pyparsing>=1.5.7',
+    # Everyting below is just for convenience while developing the GUI/viz module
```

Comment:
I would immediately put this in a `extras_require = ['gui', [list...]]` (see how it is formatted currently).

In this way you don't need to change anything.

Reply:
Done

Reply:
Well actually, I named it visualization. If you prefer gui or any other name I can change it